package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rilldata/dev",
-  "version": "0.10.0",
+  "version": "0.51.0",
```

Comment:
My comments on not relying on any `version` also apply to this root package.json. What if we set this one to `"0.0.0"` too?

Reply:
Sounds good!